dist/css/bootstrap-select.css

```diff
@@ -1,12 +1,11 @@
-/*!
- * Bootstrap-select v1.6.3 (http://silviomoreto.github.io/bootstrap-select)
- *
- * Copyright 2013-2015 bootstrap-select
- * Licensed under MIT (https://github.com/silviomoreto/bootstrap-select/blob/master/LICENSE)
- */
-
+/*!
+ * Bootstrap-select v1.6.3 (http://silviomoreto.github.io/bootstrap-select)
+ *
+ * Copyright 2013-2015 bootstrap-select
+ * Licensed under MIT (https://github.com/silviomoreto/bootstrap-select/blob/master/LICENSE)
+ */
+
 .bootstrap-select {
-  /*width: 220px\9; IE8 and below*/
```

Comment:
No changes were actually made to the source CSS as part of this PR. These changes appeared after I ran `grunt dist`. Perhaps `grunt dist` was not run for the latest merge into master?